### ISSUE TITLE
Add all instances of previously detected strings

### ISSUE THREAD

**Issue by omri374** [microsoft/presidio#859]
Presidio leverages ML models which might detect an instance in one sentence but not in another. 
By automatically adding all instances of a previously identified entity, we can increase detection recall (and potentially decrease precision)

Example (hypothetical):
> Text: "TrueForce is an American company. Lately, TrueForce was acquired by FalseForce". 

If the first "TrueForce" was detected but the second wasn't, add another `RecognizerResult` with the span of the second "TrueForce".

**Comment by navalev** [comment]
How about automaticly adding all detected entitties to a deny list, and doing a second analysis of the text with it?

**Comment by omri374** [comment]
Yep that's a good approach. We can maybe create a sample for it.
